sdk/instance/helpers.go

```diff
@@ -9,14 +10,14 @@ import (
 	"github.com/mesg-foundation/engine/service"
 )
 
-// instanceNamespace returns the namespace of the service.
-func instanceNamespace(hash hash.Hash) []string {
-	return []string{hash.String()}
+// InstanceNamespace returns the namespace of the service.
+func InstanceNamespace(hash fmt.Stringer) string {
```

Comment:
why this is exposed? maybe I'm missing something but I don't see any external package that uses it

Reply:
It was done for service logs, reverting.